### ISSUE TITLE
Question : is it python3 compatible ?

### ISSUE THREAD

**Issue by edelans** [DeaconDesperado/flask_mab#22]

**Comment by tangi75** [comment]
Not out of the box but here is how I made it run properly : 
```
pip install https://github.com/DeaconDesperado/flask_mab/archive/v1.1.1.tar.gz bunch flask-debugtoolbar
sed -i -e 's/iteritems/items/g' /opt/conda/lib/python3*/site-packages/flask_mab/debug_panels.py && sed -i -e 's/flask.ext.mab/flask_mab/g' /opt/conda/lib/python3*/site-packages/flask_mab/debug_panels.py
cd /opt/conda/lib/python3*/site-packages/flask_mab/ && mkdir -p templates/panels/ && wget -O templates/panels/mab-panel.html https://raw.githubusercontent.com/DeaconDesperado/flask_mab/master/flask_mab/templates/panels/mab-panel.html
```

**Comment by DeaconDesperado** [comment]
@tangi75 Sorry for not seeing this for so long!  I patched a 2.0 release and made it 2 and 3 compatible.
